src/lib.rs

```diff
@@ -945,21 +945,17 @@ pub unsafe extern "C" fn Reset() -> ! {
 #[link_section = ".HardFault.default"]
 #[no_mangle]
 pub unsafe extern "C" fn HardFault_(ef: &ExceptionFrame) -> ! {
-    loop {
-        // add some side effect to prevent this from turning into a UDF instruction
-        // see rust-lang/rust#28728 for details
-        atomic::compiler_fence(Ordering::SeqCst);
-    }
+    panic!("Hardfault");
```

Comment:
Nit-picking here but could this be `"HardFault"`? Documentation above would have to change too.

Reply:
Good catch, changed to this